assets/src/components/common/Pronunciation.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Refactored DefinitionPronunciation -> Pronunciation since both Definition and Conjugation could use it.